### ISSUE TITLE
GeoJSON overlay fit function as per Docs/example might be confusing.

### ISSUE THREAD

**Issue by spydmobile** [dceejay/RedMap#109]
Hi there, I spent a fair bit of time trying catch a quirk with multiple geojson overlays fighting to fit. If this helps others avoid a pitfall, then I will be happy. It was my own assumption about how fit works:

The docs and the example code  imply a boolean, so you could do something like:
```
var bb = {overlay:"Bbox", geojson:bbox, fit:false, opt:{style:aoiStyle}};
//**   == OR == **//
var bb = {overlay:"Bbox", geojson:bbox, fit:true, opt:{style:aoiStyle}};
```
But sending these to the map, does the same thing, it auto pan/zooms to the extent of the data in that layer. 

perhaps a better example than: 
```
msg.payload.command.map = {
    "overlay": "myGeoJSON",
    "geojson": { your geojson feature as an object },
    "opt": { optional geojson options, style, etc },
    "fit": true
};
```

might be:
```
msg.payload.command.map = {
    "overlay": "myGeoJSON",
    "geojson": { your geojson feature as an object },
    "opt": { optional geojson options, style, etc },
    "fit": "Set to any value this is enabled, if fit property is present it will be enabled"
};
```
or mention that to actually disable this on the layer you need to remove the **fit** property from the object entirely.  

Thanks for your awesome contribution! I use it every day.

**Comment by dceejay** [comment]
yes should check if true or "true"
will fi when back from vacation.

**Comment by spydmobile** [comment]
Super, thank you, have a great break!!!!!

**Comment by dceejay** [comment]
published as ver 2.0.19
